extensions/iam/distributed-identity/registration-service/doc/onboarding_flow.md

```diff
@@ -0,0 +1,34 @@
+## Preconditions, assumptions, terminology
+
+- It is assumed that a DID is already in place and publicly available. The generation or deployment of a DID is not part
+  of this flow.
+- The terms "connector" and "participant agent" are to be used interchangeably. Please note that both terms refer to a "
+  logical" connector - several replicas in a K8s cluster would share the same identity and thus count as one Participant
+  Agent.
+- `NewParticipant` in the diagram refers to the participant agent, which is to be onboarded in the dataspace
+- The flow presented here is _synchronous_, i.e. the verified claims are returned immediately
+- The registration service's public key is available through its DID
+- The registration service's DID key is public knowledge
+- Participant agent may be abbreviated by "PA"
+- Registration service may be abbreviated by "RS"
+
+## Onboarding flow
+
+- The new participant agent calls up the registration endpoint at `/register` and presents the following parameters:
+    - `participantAgentId`: a string uniquely identifying the PA in the dataspace
```

Comment:
Is it really the job of the participant to provide this string which must be unique within the dataspace? 
It feels to me that this unique identifier should be generated by the registration service itself, which also take care of guaranteeing the unicity of this id, of maybe I am missing something?

Reply:
Hi @bscholtes1A the pa must provide this since it is a DID